proj/src/main/webapp/signup.js

```diff
@@ -127,10 +147,11 @@ function initializePasswordValidation(){
     const noMatch = document.getElementById("noMatch");
     const btnSignUp = document.getElementById("btnSignUp");
     const pass = document.getElementById("pass");
-    const passconf = document.getElementById("passconf");
+    const passconf = document.getElementById("passconf"); 
```

Comment:
Can you remove the extra whitespace here?

Reply:
yup